repose-aggregator/src/docs/asciidoc/filters/attribute-mapping-policy-validation.adoc

```diff
@@ -17,7 +17,7 @@ This filter has no configuration.
 
 == Prerequisites & Postconditions
 === Required Request Headers
-* `Content-Type` - Declares the content type of the policy (e.g., XML or JSON) which will be used during validation.
+* `Content-Type` - Declares the content type of the policy, must be YAML.
```

Comment:
Should these be a bit more explicit and state specifically `text/yaml` since that is all that we will process?